wazo_phoned/plugins/yealink/api.yml

```diff
@@ -30,6 +31,84 @@ paths:
       - $ref: '#/parameters/Limit'
       - $ref: '#/parameters/Offset'
       - $ref: '#/parameters/AcceptLanguage'
+  /yealink/directories/lookup/{profile}:
+    get:
+      summary: Search for contacts with Yealink Phones
+      description: |
+        The `lookup` query will return a list of result matching the searched term
+        formated for Yealink Phones. The result will be retrieved from a lookup query
+        to wazo-dird.
+      operationId: yealink_lookup
+      tags:
+      - yealink
+      - directories
+      produces:
+      - text/xml
+      responses:
+        '200':
+          description: A list of contacts
+          schema:
+            $ref: '#/definitions/YealinkIPPhoneDirectory'
+        '400':
+          $ref: '#/responses/LimitOffsetInvalid'
+        '404':
+          $ref: '#/responses/ProfileInvalid'
+        '503':
+          $ref: '#/responses/AnotherServiceUnavailable'
+      parameters:
+      - $ref: '#/parameters/Profile'
+      - $ref: '#/parameters/UserUUID'
+      - $ref: '#/parameters/Term'
+      - $ref: '#/parameters/Limit'
+      - $ref: '#/parameters/Offset'
+      - $ref: '#/parameters/AcceptLanguage'
+  /yealink/users/{user_uuid}/services/dnd/enable:
+    get:
+      summary: Enable the DND service for a particular user.
+      description: |
+        This `dnd` service endpoint will enable the user DND service through
+        `wazo-confd`.
+      operationId: yealink_user_service_dnd_enable
+      tags:
+      - yealink
+      - users
+      - services
+      produces:
+      - text/xml
+      responses:
+        '200':
+          description: The status was changed
+        '400':
+          description: Provided data is incorrect or a required field is missing
+          $ref: '#/responses/InvalidDataError'
+        '404':
+          $ref: '#/responses/NoSuchUser'
+      parameters:
+      - $ref: '#/parameters/WazoUserUUID'
+  /yealink/users/{user_uuid}/services/dnd/disable:
+    get:
+      summary: Disable the DND service for a particular user.
+      description: |
+        This `dnd` service endpoint will disable the user DND service through
+        `wazo-confd`.
+      operationId: yealink_user_service_dnd_disable
+      tags:
+      - yealink
+      - users
+      - services
```

Comment:
IMO we should avoid to set more than 1-2 tag by API ... but I understand it can sometime be useful. However, I think the `services` tag is overkill and overload the spec

Personally I would only leave the yealink tag. But my comment is only to leave 2 tags max. I leave up to you to chose which to keep